store/mongo/datastore_mongo.go

```diff
@@ -350,7 +351,25 @@ func (db *DataStoreMongo) Migrate(ctx context.Context, version string) error {
 	}
 
 	for _, d := range dbs {
-		l.Infof("migrating %s", d)
+		// if not in multi tenant, then tenant will be "" and identity
```

Comment:
I'd avoid more context magic like the plague.

we're in a loop over valid tenant dbs, we have their names, migrations could just accept a verbatim dbname.

we're doing extra work by shuffling the name into a context, so that the migration can unpack it.

I won't request changes, but don't count on me doing this in any new service ;)